svir.py

```diff
@@ -1055,8 +1060,8 @@ def upload(self):
             file_size_mb, self.iface, project_definition)
         if dlg.exec_():
             project_definition['title'] = dlg.ui.title_le.text()
-            zone_label_field = dlg.ui.zone_label_field_cbx.currentText()
-            project_definition['zone_label_field'] = zone_label_field
```

Comment:
Why did you remove the selection of the zone_label_field? It is needed by the web viewer, to know which field contains the zone labels.